README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_parse_list_DOM/)
+    - [DEMO LINK](https://beliyandr.github.io/js_task_parse_list_DOM/)
```

Comment:
Add demo link to the PR description
<img width="935" alt="image" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/64750958/ab422aae-994e-41b5-ad2c-bb17ced68fdb">